Assets/Scripts/Combat/TestCombatSetup.cs

```diff
@@ -17,18 +17,19 @@ public class TestCombatSetup : MonoBehaviour
 
 	private void OnEnable()
 	{
-		Character party1 = new Character(_partyBase);
```

Comment:
I actually only learned you could do this recently, and it makes sense because since the variable is declared as a character it's obvious that the new keyword is going to make a character object
Dotnet seems to prefer it this way by default too, as long as the type is obvious